src/components/clusterprovider/minikube/minikube.ts

```diff
@@ -75,6 +78,34 @@ class MinikubeImpl implements Minikube {
     status(): Promise<MinikubeInfo> {
         return minikubeStatus(this.context);
     }
+
+    checkUpgradeAvailable() {
+        return minikubeUpgradeAvailable(this.context);
+    }
+}
+
+async function minikubeUpgradeAvailable(context: Context) {
+    if (!await checkPresent(context, CheckPresentMode.Alert)) {
+        // not installed, no upgrade.
+        return;
+    }
+
+    const sr = await context.shell.exec(`${context.binPath} update-check`);
+    const lines = sr.stdout.split(os.EOL);
+    const currentVersion = extractVersion(lines[0]);
+    const availableVersion = extractVersion(lines[1]);
+    if (currentVersion !== availableVersion) {
+        vscode.window.showInformationMessage('Minikube upgrade available.', 'Install').then((value: string) => {
```

Comment:
It might be useful to display the new version (and possibly the version the user is using?).

Reply:
done.